spec/features/ubiquity_template_work_form_spec.rb

```diff
@@ -55,7 +55,7 @@
         creator_orcid: "0000-0000-1111-2222",
         creator_institutional_relationship: "Research associate",
         creator_isni: "56273930281",
-        creator_profile_visibility: "closed"
+        creator_profile_visibility: User::PROFILE_VISIBILITY[:closed]
```

Comment:
There are no tests here for an invalid form being submitted.  Is that tested elsewhere and if so, can we add the scenario when the email is invalid to that test?